src/cairoWriter/writers/cairoFunctionDefinitionWriter.ts

```diff
@@ -40,17 +40,16 @@ export class CairoFunctionDefinitionWriter extends CairoASTNodeWriter {
     const body = this.getBody(node, writer);
 
     const returns = this.getReturns(node, writer);
-    const implicits = this.getImplicits(node);
 
     return [
-      [documentation, ...decorator, `func ${name}${implicits}(${args})${returns}{`, body, `}`]
+      [documentation, ...decorator, `fn ${name}(${args})${returns}{`, body, `}`]
         .filter(notNull)
         .join('\n'),
     ];
   }
 
   private getDecorator(node: CairoFunctionDefinition): string[] {
-    if (node.kind === FunctionKind.Constructor) return ['@constructor'];
+    if (node.kind === FunctionKind.Constructor) return ['#[constructor]'];
     const decorators: string[] = [];
     if (node.kind === FunctionKind.Fallback) {
       decorators.push('@raw_input');
```

Comment:
@rjnrohit commented on the original PR https://github.com/NethermindEth/warp/pull/969#discussion_r1125955293

Reply:
Done!